packages/agents/router/src/subscriber/subscriber.ts

```diff
@@ -93,10 +98,24 @@ export const makeSubscriber = async (_configOverride?: NxtpRouterConfig) => {
 
     /// MARK - Cold Start Housekeeping
     try {
-      const res = await axios.get(`${context.config.sequencerUrl}/ping`);
-      context.logger.info("Ping response received from sequencer", requestContext, methodContext, {
+      const res = await axios.get(`${context.config.sequencerUrl}/supportedBidVersion`);
+      context.logger.info("supportedBidVersion response received from sequencer", requestContext, methodContext, {
         response: res.data,
       });
+
+      const supportedBidVersion: string = res.data;
+      // check if bid router version is compatible with hosted sequencer
+      const checkVersion = compare(version, supportedBidVersion, "<");
+      if (checkVersion) {
+        context.logger.error(
+          "Invalid Bid Version, please update router! Exiting :(",
```

Comment:
It's not technically bid version, it's sequencer version.

Reply:
it's supportedBidVersion, In the sequencer config, we set the supported version so that the router version needs to be >= supportedBidVersion.